package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/sqnc-process-management",
-  "version": "2.2.16",
```

Comment:
Minor rather than patch?

Reply:
doesn't change the interface, it only changes performance. This is a patch